Dockerfile

```diff
@@ -2,11 +2,11 @@ FROM percona:5.6
 
 RUN apt-get update \
     && apt-get install -y \
-    python \
-    python-dev \
-    gcc \
-    curl \
-    percona-xtrabackup \
+        python \
+        python-dev \
+        gcc \
```

Comment:
I'd love to adding the `pip install` steps in here so we can `apt-get purge` GCC after it's done its job, similar to https://github.com/autopilotpattern/nfsserver/blob/wip/Dockerfile#L8-L19.